store/user/dashboard.js

```diff
@@ -15,6 +25,58 @@ class DashboardUser extends User {
       (e) => e.name.toLowerCase().search(searchTerm.toLowerCase()) !== -1
     )
   }
+
+  onDataProviderChange(callback) {
```

Comment:
The method is not clear, it's better to have `addEventListener` or `on`, or extend EventTarget somewhere and use its API. However, we can keep this for now.

Reply:
If you wish we can create some global store event bus.

Reply:
I wanted to but I did not because it was not necessary

Reply:
All right I think it will be better with the event system. Let me change it.

Reply:
[`a3d3422` (#382)](https://github.com/oacore/dashboard/pull/382/commits/a3d342287a793bd52fe3f55652efe3339bce68e8)